plugins/uefi/efi/meson.build

```diff
@@ -135,20 +135,15 @@ so = custom_target('fwup.so',
                              efi_ldflags + ['@INPUT@'] +
                              ['-lefi', '-lgnuefi', libgcc_file_name])
 
+build_tool = join_paths(meson.source_root(), 'plugins', 'uefi', 'efi', 'generate_binary.sh')
+build_cmd = [build_tool, '@INPUT@', '@OUTPUT@', efi_format, objcopy]
+if genpeimg.found()
+  build_cmd += [genpeimg]
```

Comment:
I realize that this is going to break for `if efi_arch == 'aarch64' or efi_arch == 'arm'`